tasks/task12.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Soret effect\n",
+    "\n",
+    "This task will demonstrate how to setup a FESTIM model including the Soret effect.\n",
+    "\n",
+    "We will model a 6 mm slab in steady state with $c=0$ on the boundaries and a volumetric source of particle representing a plasma implantation in the first few nanometres.\n",
+    "\n",
+    "## FESTIM model"
```

Comment:
Maybe, it worth showing how the flux changes when we include the Soret effect, so there won't be a need to check the theory section?

Reply:
Sure yes I can add this

Reply:
I added this + a new plot for the T profile. Let me know what you think